plugins/task-resources/src/components/kanban/KanbanPanelEmpty.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+    import { IconAdd, Button, EditBox, ActionIcon, IconClose } from '@anticrm/ui'
+    import { createEventDispatcher } from 'svelte';
+    import tracker from '../../plugin';
+
+    let isAdding = false
+    const dispatch = createEventDispatcher()
+    const newList = {
+        title: '',
+        color: ''
+    }
+
+    async function onAdd () {
+        if (!newList.title) return;
+        dispatch('add', newList)
+        isAdding = false
```

Comment:
Should we set title to '' after value is added?

Reply:
Set